sdcm/ec2_client.py

```diff
@@ -376,26 +391,31 @@ def get_instance_by_private_ip(self, private_ip):
 
 
 if __name__ == '__main__':
+    import unittest
```

Comment:
remove this test code

Reply:
move this to test_manual.py, i.e. test we might need for some type of integration while developing